x-pack/legacy/plugins/apm/public/components/shared/charts/TransactionCharts/RegionMapChart/ColorProgressionBar.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+export const ColorProgressionBar: React.SFC<{
```

Comment:
Maybe simply call this `Legend` since it's not really a reusable component but specifically designed for the purpose of being a legend.